stan/math/rev/mat/fun/gp_periodic_cov.hpp

```diff
@@ -350,7 +350,7 @@ gp_periodic_cov(const std::vector<T_x> &x, const var &sigma, const var &l,
  *   x is nan or infinite
  */
 template <typename T_x>
-inline typename std::enable_if<
+inline typename boost::enable_if<
```

Comment:
?

Reply:
lol oh nvm

Reply:
This is a deliberate error commit to trigger the error message :) See the post on the PR thread in 5 seconds :P